src/components/DacUsers.js

```diff
@@ -0,0 +1,85 @@
+import _ from 'lodash';
+import { Component } from 'react';
+import { a, div, hh } from 'react-hyperscript-helpers';
+import { CHAIR, MEMBER } from './modals/AddDacModal';
+
+
+const buttonPadding = { paddingTop: 6 };
+const headerStyle = { fontWeight: 500, color: '#00609f' };
+
+export const DacUsers = hh(class DacUsers extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      dac: props.dac,
+      removeButton: props.removeButton,
+      removeHandler: props.removeButton ? props.removeHandler : () => {},
+      removedIds: []
+    };
+    this.onRemove = this.onRemove.bind(this);
+    this.columnClass = this.columnClass.bind(this);
+    this.makeRow = this.makeRow.bind(this);
+  }
+
+  onRemove(dacId, dacUserId, type) {
```

Comment:
Just confirming: this works like a toggle ? if dacUserId exists in  removedIds, remove it from removedIds, otherwise add it. ?

Reply:
It does function like a toggle, yes. This allows the component to maintain who is selected for removal and who isn't. Those that are removed can be re-added (with same role or different, doesn't matter) and those who are not removed (i.e. they are unchanged), cannot be re-added with a different role.